packages/sources/finage-test/src/config/index.ts

```diff
@@ -0,0 +1,40 @@
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'API endpoint for Finage',
+    default: 'https://api.finage.co.uk',
+    type: 'string',
+  },
+  API_KEY: {
+    type: 'string',
+    required: true,
+    sensitive: true,
+    description: "An API key that can be obtained from the data provider's dashboard",
+  },
+  WS_SOCKET_KEY: {
+    type: 'string',
+    required: false,
```

Comment:
Based on v2 I think `WS_SOCKET_KEY` should be required.
```suggestion
    required: true,
```